Gemfile.lock

```diff
@@ -106,11 +106,11 @@ GEM
     bindex (0.8.1)
     binding_of_caller (1.0.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.7.2)
+    bootsnap (1.7.3)
       msgpack (~> 1.0)
-    bootstrap (5.0.0.beta1)
+    bootstrap (5.0.0.beta2)
```

Comment:
### [bootstrap](https://github.com/twbs/bootstrap-rubygem)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 5.0.0.beta1 → 5.0.0.beta2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/twbs/bootstrap-rubygem/compare/v5.0.0.beta1...v5.0.0.beta2).

These are the individual commits:
- (615e415) [Bump popper_js version](https://github.com/twbs/bootstrap-rubygem/commit/615e415ec4c4de8852bbc826f78d881a845e8da0)
- (083f8d2) [Bump to v5.0.0.beta2](https://github.com/twbs/bootstrap-rubygem/commit/083f8d22a9c4231517ea0e5c9e2500cd14cf6901)
- (db61cec) [JS: Define components in the `bootstrap` namespace](https://github.com/twbs/bootstrap-rubygem/commit/db61cec2086ab576efc3e328b4e8a5fd06094bff)
- (c43a4cb) [Update test rails app view](https://github.com/twbs/bootstrap-rubygem/commit/c43a4cb312942449be30a3d0abfe324dc9ae6980)
- (e9fb4c7) [travis: Add Rails 6.1](https://github.com/twbs/bootstrap-rubygem/commit/e9fb4c7bc977744a16e74e9133aea0f07e4953c2)
- (8b5b579) [Bump minimum Popper version to v2.8.6](https://github.com/twbs/bootstrap-rubygem/commit/8b5b57938f0fba0a0eb7d190db2c28ec9848130d)


</details>